src/main/java/com/launchacademy/giantleap/controllers/ReviewsRestController.java

```diff
@@ -11,11 +12,11 @@
 
 @RestController
 @RequestMapping
-public class ReviewRestController {
+public class ReviewsRestController {
```

Comment:
this file should be located in `controllers/api/v1`

Reply:
Clarifying question . . . all rest controllers should be located in the `controllers/api/v1`, not just this file (i.e. ReviewsRestController), correct?